ci/build.yml

```
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.21.3-bullseye
+    tag: 1.22-bullseye
```

we usually specify patch version as well. 
```suggestion
    tag: 1.22.2-bullseye
```